ang/crmCaseType.js

```diff
@@ -169,7 +178,10 @@
         $scope.caseType = apiCalls.caseType;
       } else {
         // new case type
+        var defaultCategory = _.find($scope.caseTypeCategories, { name: 'WORKFLOW' }) || {};
+
         $scope.caseType = _.cloneDeep(newCaseTypeTemplate);
+        $scope.caseType.category = defaultCategory.value;
```

Comment:
What if default category is not found? there will be an error then. Should not we add conditions?

Reply:
There should not, since `defaultCategory` would be an empty object:

```js
var defaultCategory = _.find($scope.caseTypeCategories, { name: 'WORKFLOW' }) || {};
```

This would set `category` as undefined without throwing an error since in JS accessing a non defined property of an object just return undefined (the problem would be to access a property of an undefined value).

Also, workflow is defined as `is_reserved = true` on the API, that means it can't be deleted.